.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,100 @@
+name: End-to-End Tests
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+    branches: [trunk]
```

Comment:
Does this run on all pushes to all PRs/branches? I'm thinking that's what we want, so this is another check against PRs as you work.

Reply:
Wahey! I see this is running against this PR, I think `pull_request` is doing what we need 🌮 

<img width="1034" alt="Screen Shot 2020-12-22 at 12 58 32 PM" src="https://user-images.githubusercontent.com/4167300/102833262-7887b280-4455-11eb-9cca-fcfca51c35c2.png">

Reply:
Yep! This will run every time something is pushed to a branch with an open PR to `trunk` and it will also run when any code is pushed to `trunk` (so when the PR is merged I guess these checks will run again)